src/resources/postcss/tickets-commerce/_checkout.pcss

```diff
@@ -149,12 +149,43 @@
 		max-width: 85px;
 	}
 
+	.tribe-tickets__commerce-checkout-cart-item-price {
+		.tec-tickets-price {
```

Comment:
The naming does not follow BEM conventions, and in theory, we should also have classes at the same level, not nested (for this, and the following definitions).

Reply:
@juanfra I agree that they aren't BEM conventions. These are classes that have been in place for awhile, now. They aren't new. I felt the nesting was necessary since this same template is used in different areas and we wanted to differentiate the cart items from other areas.